docs/patterns/clustered-listening-tentacles.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Clustered Listening Tentacles
+description: Configuring the Octopus Tentacle in an Active/Passive cluster. 
+position: 0
+---
+
+You can configure a pair of Octopus Tentacles in an Active/Passive failover cluster using shared storage with the **Failover Cluster Manager**. This approach has been tested on Windows Server 2016. 
+
+To perform this task, first ensure that a shared drive is available to the node where you are installing the new Octopus Tentacle instance. Once all nodes have the Octopus Tentacle instances installed, pick any node to generate and export a private key. Proceed to import the new private key onto every Octopus Tentacle instance, including the node which was used to export it. 
+
+Using the Failover Cluster Manager you then configure the Octopus Tentacle instance service as a generic service, ensuring it has a clustered IP and gets registered in DNS. Once the Tentacles have been setup, use the Octopus Server to discover the new Tentacle cluster on the Clustered IP address specified when making Octopus Tentacles a generic service cluster. 
+
+:::warning
+**Shared Storage Considerations**
+It is not possible to store the **tentacle.config** file in shared storage because the Tentacle.Certificate component gets encrypted using a node's machine-specific key. If you attempt to store the tentacle.config file in shared storage, you will encounter this error: **Key not vaild for use in specified state** upon switching to a new active node. This occurs because the new active node is not able to decrypt the private key which will result in the Tentacle service failing to start.
```

Comment:
The error message should be surrounded by backticks ` rather than bold.

` which will result in the Tentacle service failing to start` => `, resulting in the Tentacle service failing to start`